src/marqo/s2_inference/processing/video.py

```diff
@@ -0,0 +1,28 @@
+import moviepy
+import PIL
+
+def convert_to_listofPIL(video):
+    frames = list(video.iter_frames())
+    return [PIL.Image.fromarray(i.astype("uint8"), 'RGB') for i in frames]
+
+def chunk_video(field_content : str, device = "cpu", method = "simple"):
+    clip = field_content
+    duration = clip.duration
+    content_chunks = []
+    reminder = duration % 10
+
+    if duration < 15:
```

Comment:
how are these values (`15`) determined? we might want to have them as keyword args or set them in a config somewhere

Reply:
There are 2 ways to set this duration parameter: 
1. We can create a new `IndexSettingsField.video_preprocessing` dictionary in configs.py, that can have a `video_chunk_length` parameter (or Jesse can come up with a better name for this param)

See here for reference: https://github.com/marqo-ai/marqo/blob/4a6dfd7a5170336aafd472092bc5c9828a0c3b7a/src/marqo/tensor_search/configs.py#L19

2. If you think the user can crash Marqo with an overly massive video chunk you define it with a default environment variable. 

- Default env vars defined here:  https://github.com/marqo-ai/marqo/blob/4a6dfd7a5170336aafd472092bc5c9828a0c3b7a/src/marqo/tensor_search/configs.py#L27
- To read the environment variable, use this function: https://github.com/marqo-ai/marqo/blob/4a6dfd7a5170336aafd472092bc5c9828a0c3b7a/src/marqo/tensor_search/utils.py#L164

Reply:
The length of each chunk is 10s as the x_clip model is trained by videos with 10 seconds.
    
    
    The last chunk is very likely to be less than 10s, we solve the problem using the following rules:
    If the last chunk is longer than 5 seconds, we keep the chunk.
    If the last chunk is less than 5 seconds, we add this chunk to the previous one.

    For example, for a video with 34s, we will have 3 chunks with length [10s, 10s, 14s]
    As for a video with 38s, we will have 4 chunks with length [10s, 10s, 10s, 8s]

Reply:
I will add two configs, `video_chunk_length` and `video_chunk_merge_flag` to solve this problem.